src/commands/profile/decrypt.ts

```diff
@@ -34,8 +36,19 @@ export default class extends ProfileCommand {
     async execute(options: ProfileOptions) {
         const profile = this.getProfile(options)
         const password = await new PasswordResolver().resolve(options)
-        const account = profile.decrypt(password)
-        const text = new AccountCredentialsTable(account).toString()
-        console.log(text)
+
+        if (profile.type === 'PrivateKey') {
+            const account = profile.decrypt(password)
+            console.log(AccountCredentialsTable.createFromAccount(account).toString())
+        } else {
+            if (!profile.encryptedPassphrase || !profile.pathNumber) {return }
```

Comment:
In this case, the command could  print a message instead of returning an empty response.

Reply:
It's just a trick for typing, this case shouldn't happen

Will modify it